app/public/css/styles.css

```diff
@@ -1477,3 +1476,65 @@ p {
   background-position: center;
   background-image: url('../assets/input_pencil_icon.svg');
 }
+
+/* CARD */
+
+.card {
+  background-color: white;
+  padding: 40px 56px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
+}
+
+/* TRANSACTIONS */
+
+.transactions {
+  padding: 40px 56px;
+}
+
+.transactions-title {
+  margin-bottom: 24px;
+}
+
+.transaction-item {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  padding-top: 24px;
+  padding-bottom: 24px;
+  border-top: 1px solid var(--color-border);
+  color: var(--color-grey);
+}
+
+.transaction-date {
+  display: flex;
```

Comment:
This is not the proper way how to do it if you use the *grid*. The grid supports `align-self: center;` for vertical alignment in a grid item.